### DISCUSSION TITLE
add option to redirect calls to contacts to apps (selected by user) by default

### DISCUSSION THREAD

**Discussion by drugdeliverthatwantprivacy** [FossifyOrg/General-Discussion#543] [Polls]
Normal calls do not have end-to-end encryption, so your ISP, government could monitor your conversations, it would be nice to improve user privacy if a function was added to redirect calls to contacts to another application (where it is possible to select the applications, it would be nice if a text appears in the function saying that for this function applications with end-to-end encryption are recommended to improve the privacy of communications, example: WhatsApp, Signal...) (adding this text would be nice so that users with less technical knowledge understand the usefulness of this function)

**Comment by naveensingh** [comment]
Doesn't sound like a good idea to me but if you'd like this to be considered in the future, consider creating a feature request for it.
